starlight/star_commands/help_command/injector.py

```diff
@@ -73,6 +87,18 @@ def __init__(self, inject: commands.HelpCommand, *args: Any, **kwargs: Any) -> N
             inject.__original_callback__.callback, globals(), skip_parameters=1  # type: ignore
         )
 
+        # copy over descriptions to each command
+        descriptions: Dict[str, str] = getattr(inject, "__starlight_help_parameter_descriptions__", {})
+
+        app_commands.describe(**descriptions)(self.app_command)
```

Comment:
`self.app_command` errors for `with_app_command=False`. You can just set it to `self` since it's a hybrid darling

Reply:
nvm, after seeing the behaviour of how helpcommand deal with that, keep it as is. forget what I said